pysatMissions/instruments/methods/orbits.py

```diff
@@ -0,0 +1,139 @@
+"""Methods to convert orbital information for propagators."""
+
+import numpy as np
+import warnings
+
+
+def _check_orbital_params(kwargs=None):
+    """Check that a complete set of unconflicted orbital parameters exist.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Dictionary of optional kwargs passed through upon initialization
+        of pysat instrument.
+
+    """
+
+    elements = list(kwargs['load'].keys())
+
+    keplerians = ['alt_periapsis', 'inclination']
+    tles = ['TLE1', 'TLE2']
+    errmsg = 'Insufficient kwargs.  Kwarg group requires {:}'
+    for group in [tles, keplerians]:
+        bools = [v in elements for v in group]
+        # Check if group is incomplete.
+        if any(bools) and not all(bools):
+            raise KeyError(errmsg.format(', '.join(group)))
+    if all(v in elements for v in tles) and all(v in elements for v in keplerians):
+        warnings.warn(' '.join(['Cannot use both Keplerians and TLEs.',
+                                'Defaulting to Keplerians.']))
+
+    return
+
+
+def _get_constants(planet='Earth'):
+    """Retrieve planetary constants for calculations.
+
+    Parameters
+    ----------
+    planet : str
+        The name of the planet of interest.
+        (default='Earth')
+
+    Returns
+    -------
+    radius : float (km)
+        The average radius to the surface of a planet.
+    mass : float (kg)
+        The average mass of the planet.
+    gravity : float (m**3 kg / s**2)
+        Newton's gravitational constant
+    """
+
+    radius = {'Earth': 6371.2}
+    mass = {'Earth': 5.9742e24}
```

Comment:
Is there an issue to add other planets?

Reply:
Also, recommend using a lowercase string so you can cast it to lower and allow various capitalization schemes as input.

Reply:
Added #76